File: Gyak3/3het2.cpp
#include <iostream>
#define SZAMOKSZAMA 10
using namespace std;

int main() { 
    int szamok[SZAMOKSZAMA];
    double osszeg=0;
    cout << "Irjon be 10 szamot: " << endl;
    for (int i = 0; i < 10; i++)
    {
        cout << i+1 << ". szam: "; cin >> szamok[i];
        osszeg += szamok[i];
    }
    for (int j = 0; j <= 9; j++)
    {
        if(szamok[j] < osszeg/SZAMOKSZAMA) cout << szamok[j] << " az atlag alatt van.\n";
    }
    return 0;
}

File: Gyak3/3het2.py
import numpy as np

SZAMOKSZAMA = 10

def main():
    szamok = [0.0]*10
    osszeg = 0.0
    print("Irjon be 10 szamot: ")
    for i in range(0,10):
        szamok[i] = float(input(f"{i+1}. szam: "))
        osszeg+=szamok[i]
    for item in szamok:
        if item < osszeg/SZAMOKSZAMA:
            print(f"{item} az atlag alatt van.")

main()